packages/quantic/force-app/main/default/lwc/quanticResultAction/quanticResultAction.js

```diff
@@ -38,6 +38,7 @@ export default class QuanticResultAction extends LightningElement {
    * The result to perform the action on.
    * @api
    * @type {Result}
+   * @deprecated
```

Comment:
You can add some details after `@deprecated` maybe to explain the deprecation or to say "deprecated since version X"

Reply:
sure yeah 👌